rust/src/action.rs

```diff
@@ -48,6 +48,67 @@ fn gen_action_type_error(action: &str, field: &str, expected_type: &str) -> Acti
     ))
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+#[serde(untagged)]
+pub enum ColumnValueStat {
+    Column(HashMap<String, ColumnValueStat>),
+    Value(serde_json::Value),
```

Comment:
leaving the value as generic json value for now since the implementation is not released in open-source code base and the official spec is ambiguous on what types are supported. We can refactor this once reference implementation is released.